scripts/etl/extract.py

```diff
@@ -1 +1,19 @@
-# Extract datasets from sources
\ No newline at end of file
+# Extract datasets from sources
+
+import PyPDF2
+
+# Path to the uploaded PDF file
+pdf_path = "../../therapists_guide_to_brief_cbtmanual.pdf"
+txt_output_path = "../../therapists_guide_to_brief_cbtmanual.txt"
+
+# Extract text from the PDF
+with open(pdf_path, "rb") as pdf_file:
+    reader = PyPDF2.PdfReader(pdf_file)
+    extracted_text = "\n".join([page.extract_text() for page in reader.pages if page.extract_text()])
+
+# Save the extracted text to a .txt file
+with open(txt_output_path, "w", encoding="utf-8") as txt_file:
+    txt_file.write(extracted_text)
+
+# Provide the file path for user download
+txt_output_path
```

Comment:
One of the coding best practices we need to follow is to not have any dangling lines of code outside of functions/classes. If this is a standalone script, can we put this in a `main` function with an `if __name__=='__main__'` block? Also, that last line doesn't appear to be doing anything (it's more of a notebook thing to get Jupyter to print a value). We can probably remove it.